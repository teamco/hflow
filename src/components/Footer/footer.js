import React from 'react';
import { Layout } from 'antd';
import classnames from 'classnames';

import styles from '@/components/Footer/footer.module.less';

export const Footer = props => {
  const { children, className, ...rest } = props;

  return (
      <Layout.Footer className={classnames(styles.footer, className)}
                     data-testid={'footer'}
                     {...rest}>
        {children}
      </Layout.Footer>
  );
};
