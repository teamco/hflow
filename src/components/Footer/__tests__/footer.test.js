import React from 'react';
import { cleanup } from '@testing-library/react';

import { Footer } from '../footer';

import { expectations } from '__tests__/helper';

describe('@/components/Footer', () => {

  // Note: running cleanup afterEach is done automatically for you in @testing-library/react@9.0.0 or higher
  // unmount and cleanup DOM after the test is finished.
  afterEach(cleanup);

  it('Footer', () => {
    const _footerDom = expectations(Footer, 'footer', {});

    expect(_footerDom).toHaveClass('ant-layout-footer footer');
  });
});
