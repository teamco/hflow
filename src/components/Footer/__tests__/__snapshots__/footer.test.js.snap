// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/Footer Footer 1`] = `
<DocumentFragment>
  <footer
    class="ant-layout-footer footer"
    data-testid="footer"
  />
</DocumentFragment>
`;

exports[`components/Footer Footer 1`] = `
<DocumentFragment>
  <footer
    class="ant-layout-footer footer"
    data-testid="footer"
  />
</DocumentFragment>
`;
